include/luabind/type_storage.hpp

```diff
@@ -36,6 +36,8 @@ struct type_info {
     lua_CFunction new_index;
     lua_CFunction subscript_read;
```

Comment:
subscript_read/write are the leftovers from my attempt at implementing array access, please remove those.

Reply:
Done